e2e/sample.config.extended.yaml

```diff
@@ -40,7 +42,7 @@ activeRelayer: hermes # override with RELAYER_ID
 relayers:
   - id: hermes
     image: ghcr.io/informalsystems/hermes
-    tag: "1.10.4" # override with RELAYER_TAG
+    tag: "1.10.0" # override with RELAYER_TAG
```

Comment:
hermes tag can remain
```suggestion
    tag: "1.10.4" # override with RELAYER_TAG
```

Reply:
Yup! Should have been fixed all across the line now